src/runtime/components/overlays/Slideovers.client.vue

```diff
@@ -1,18 +1,19 @@
 <template>
   <component
-    :is="slideoverState.component"
-    v-if="slideoverState"
-    v-bind="slideoverState.props"
-    v-model="isOpen"
-    @after-leave="reset"
+    :is="instance.component"
+    v-for="(instance, index) in slideoverInstances"
+    :key="index"
```

Comment:
I'd suggest using the instance ID. Also, you update the list using `filter` and reinserting the new element at the end, which will change the item's indexes.

```suggestion
    :key="instance.id"
```